api/oauth/post.ts

```diff
@@ -40,11 +47,19 @@ const logic = async (
         },
       }
     )
-    .catch((e) =>
-      Promise.reject(
+    .catch((e) => {
+      console.error("Failed to fetch access token");
+      console.error(e);
+      console.error("Arguments:");
+      const maskedArgs = {
+        ...args,
+        code: maskString(args.code),
+      };
+      console.error(maskedArgs);
+      return Promise.reject(
         new Error(`Failed to get access token: ${e.response.data}`)
-      )
-    );
+      );
+    });
   const { access_token } = data;
   const privateKey = process.env.APP_PRIVATE_KEY;
   const workspace = privateKey
```

Comment:
let's do the same thing on the Octokit api call below

Reply:
nice call 🙌

![image](https://github.com/samepage-network/github-samepage/assets/3792666/88865b65-2dae-42e6-8427-fe0c7016aa2c)